src/ontology/mondo-edit.obo

```diff
@@ -412736,6 +412983,41 @@ synonym: "SPGF53" EXACT [OMIM:619258]
 xref: OMIM:619258 {source="MONDO:equivalentTo"}
 is_a: MONDO:0004983 {source="OMIM:619258"} ! azoospermia
 
+[Term]
+id: MONDO:0030990
+name: Kohlschutter-Tonz syndrome-like
+synonym: "KTZSL" EXACT ABBREVIATION [OMIM:619229]
+xref: OMIM:619229 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+
+[Term]
+id: MONDO:0030991
+name: bile acid conjugation defect 1
+synonym: "BACD1" EXACT ABBREVIATION [OMIM:619232]
+xref: OMIM:619232 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```

Comment:
guess there is no BACD2?

Reply:
right, it doesn't look like there is:
https://omim.org/search?index=entry&start=1&limit=10&sort=score+desc%2C+prefix_sort+desc&search=BILE+ACID+CONJUGATION+DEFECT+